Rubycat/proveit/_meta/fields.yml

```
@@ -0,0 +1,20 @@
+proveit.source.type:
+  type: string
+  name: proveit.source.type
+  description: Type of the source
+proveit.source.componentId:
+  type: string
+  name: proveit.source.componentId
+  description: Source Component Identifier
+proveit.source.roles:
+  type: array
+  name: proveit.source.roles
+  description: User roles
+proveit.source.profiles:
+  type: array
+  name: proveit.source.profiles
+  description: User profiles
+proveit.context.auth_mode:
+  type: string
+  name: proveit.context.auth_mode
+  description: Authentication mode on the service
```

For custom fields, the practice is to prefix with the name of the vendor and the name of the product
```suggestion
rubycat.proveit.source.type:
  type: string
  name: rubycat.proveit.source.type
  description: Type of the source
rubycat.proveit.source.componentId:
  type: string
  name: rubycat.proveit.source.componentId
  description: Source Component Identifier
rubycat.proveit.source.roles:
  type: array
  name: rubycat.proveit.source.roles
  description: User roles
rubycat.proveit.source.profiles:
  type: array
  name: rubycat.proveit.source.profiles
  description: User profiles
rubycat.proveit.context.auth_mode:
  type: string
  name: rubycat.proveit.context.auth_mode
  description: Authentication mode on the service
```